src/Console/Commands/PostmanCollection.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+namespace HZ\Illuminate\Mongez\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class PostmanCollection extends Command
+{
+    /**
+     * Path Folder of modules
+    */
+    const MODULE_PATH = 'app/Modules';
+
+    /**
+     * Docs Folder Name
+    */
+    const DOCS_FOLDER_NAME = 'docs';
+
+    /**
+     * default generate file name
+    */
+    const DEFAULT_GENERATE_FILE_NAME = 'project-api-collection';
+
+    /**
+     * Postman Files.
+     *
+     * @var array
+     */
+    private $postmanFiles = [];
+
+    /**
+     * Content of each postman file module.
+     *
+     * @var array
+     */
+    private $filesContent = [];
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'engez:postman {--fileName=}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generate Root Collection ..';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $this->loadPostmanFiles();
+
+        $this->loadContentPostmanFiles();
+
+        $this->createCollection();
+
+        return 0;
+    }
+
+    /**
+     * load all postman files 
+     * 
+     * @return void
+    */
+    private function loadPostmanFiles() 
+    {
+        $modulePath = $this->getModulePath();
+
+        $this->postmanFiles = array_filter(glob($modulePath . '/*/' . self::DOCS_FOLDER_NAME . '/*.postman.json'));
+
+        $this->info('Success Load Postman Files ..');
+    }
+
+    /**
+     * Load Content of postman file
+     * 
+     * @return void
+    */
+    private function loadContentPostmanFiles()
+    {
+        foreach($this->postmanFiles as $postmanFile) {
+            $content = $this->pareFileToJson($postmanFile);
+
+            $name = $content['info']['name'];
+            
+            $this->filesContent[] = $content;
+
+            $this->info("Collection [ $name ] was successfully added.");
+        }
+    }
+
+    /**
+     * Create Root Collection
+     * 
+     * @return void
+    */
+    private function createCollection()
+    {
+        $appName = env('APP_NAME', 'Postman Collection with Mongez');
+
+
+        $collection = [
+            'item' => [],
+            'variable' => [],
+            'info' => [
+                'name' => $appName,
+                'schema' => 'https://schema.getpostman.com/json/collection/v2.1.0/collection.json'
+            ]
+        ];
+
+        foreach($this->filesContent as $content) {
+            // Remove Content of Globel Variable in each Collection  
```

Comment:
Perfectly written, i wouldn't get it if you didn't write these two lines (Y)

Reply:
Thanks, it was written by me.